lib/puppet/indirector/json.rb

```diff
@@ -52,7 +52,7 @@ def load_json_from_file(file, key)
     json = nil
 
     begin
-      json = File.read(file)
+      json = File.read(file).force_encoding(Encoding::UTF_8)
```

Comment:
So the read_preserve actually checks to see if the encoding is valid, would it be even with bad characters in this case? @joshcooper I think I understand that you are saying without saving it in the proper encoding it will come back with a valid encoding incorrectly.